webhook/resourcesemantics/validation/reconcile_config.go

```diff
@@ -103,7 +103,19 @@ func (ac *reconciler) reconcileValidatingWebhook(ctx context.Context, caCert []b
 				APIVersions: []string{gvk.Version},
 				Resources:   []string{plural + "/*"},
 			},
-		})
+		}
+
+		// Add supported verbs from callbacks to the rule operations
+		if callback, ok := ac.callbacks[gvk]; ok {
```

Comment:
As I said in chat, I'd like to see us address the whole problem here, and not just have a band-aid.  We should expand when `Validate` is called to include Delete, improve the context tagging, and ensure `oldObj` is the receiver to Validate calls.

Reply:
I think this is a step in the right direction.

Here is where we decode oldObj for the delete action:
https://github.com/knative/pkg/blob/4e57475bc87c1aba3e7fcc0207b593dc7186eb83/webhook/resourcesemantics/validation/validation_admit.go#L185

Reply:
https://github.com/knative/pkg/pull/1219

I think this is the rest of what we want here

Reply:
Thank you @whaught! @mattmoor would #1219 + this PR be sufficient?

Reply:
looks like I hadn't hit submit on my comments on the other PR 🤦 

I think the other PR makes a majority of this PR not necessary.  I think the only bit it's missing is updating the webhook to include Delete, which can be simpler than what's in this PR because this is dynamic and with the other PR it can still be a fixed set.

Reply:
That sounds good to me, the other PR can just add the Delete operation statically